WordPress/Jetpack/Resources/AppStoreStrings.po

```diff
@@ -0,0 +1,109 @@
+# Translation of Release Notes & Apple Store Description in English (US)
```

Comment:
To generated this file:

1. I copied the WordPress one
2. I deleted the irrelevant `msgctxt`+`msgid`+`msgstr` groups
3. I ensured all the `msgid` values matched what's currently on GlotPress

I then run the lane and verified that the original `msgstr` values from WordPress were replaced with the content of the Jetpack `.txt` files as configured by the lane.